sections/featured-product.liquid

```diff
@@ -5,6 +5,9 @@
 {{ 'component-deferred-media.css' | asset_url | stylesheet_tag }}
 {{ 'component-rating.css' | asset_url | stylesheet_tag }}
 {{ 'component-volume-pricing.css' | asset_url | stylesheet_tag }}
+{{ 'component-product-variant-picker.css' | asset_url | stylesheet_tag }}
+{{ 'component-swatch.css' | asset_url | stylesheet_tag }}
+{{ 'component-swatch-input.css' | asset_url | stylesheet_tag }}
```

Comment:
Might be worth wrapping in a conditional if the product doesn't have variants 🤔 Or something to track for later follow ups. 
On the product object there is `has_online_default_variant` which we could use so we don't load unused styling ([docs](https://shopify.dev/docs/api/liquid/objects/product#product-has_only_default_variant)).

Reply:
When there is more than one Featured Product section, we will likely render the file reference more than once anyway.  But that said, I think it's okay to not check for whether or not it's configured.